lib/oli_web/components/delivery/discussion_activity/discussion_activity_live.ex

```diff
@@ -0,0 +1,254 @@
+defmodule OliWeb.Components.Delivery.DiscussionActivityLive do
+  use Surface.LiveView
+  use OliWeb.Common.Modal
+
+  alias OliWeb.Common.PagedTable
+  alias OliWeb.Discussion.TableModel, as: DiscussionTableModel
+  alias OliWeb.CollaborationLive.InstructorTableModel, as: CollabSpaceTableModel
+  alias Oli.Resources.Collaboration
+  alias OliWeb.Common.Confirm
+  alias OliWeb.Common.Table.SortableTableModel
+
+  data posts, :list, default: []
+  data discussion_table_model, :struct
+  data collab_space_table_model, :struct
+  data filter, :string, default: "all"
+  data limit, :number, default: 10
+  data offset, :number, default: 0
+
+  def mount(_params, %{"section_slug" => section_slug}, socket) do
```

Comment:
I think it would be great to have `filter` `limit` and `offset` in the url params, to keep the current filter applied after page refreshing (and allowing the user to bookmark the url with the applied filters), WDYT?